tests/test_autoreport.py

```diff
@@ -86,13 +85,13 @@ def test_simple_report(self):
                 'nom du restaurant',
                 {'frequency': [('Taco Truck', 1),
                                ('Harvest', 1),
-                               ('Los pollos hermanos', 1),
-                               ('Wololo', 1)],
+                               ('Wololo', 1),
+                               ('Los pollos hermanos', 1)],
```

Comment:
I don't remember where exactly, but because of the usage  of `defaultdict` when getting the stats.
Python3 and Python2 did not return the list in the same order and the test failed one version or the other.
That's why I've created this `OrderedDefaultdict` class which keeps the same order whatever the version.

Reply:
aha, now it matches the submission order :+1: